config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hazelcast-configuration
+data:
+  hazelcast.yaml: |-
+    hazelcast:
+      network:
+        join:
+          multicast:
+            enabled: false
+          kubernetes:
+            enabled: true
+        rest-api:
```

Comment:
Out of curiosity, is that required for k8s deployments or just an optional feature? I see corresponding parts at hazelcast.yaml but I wonder if they are being used by kubernetes service or some other.

Reply:
Doesn't seem to be used. I will remove it, as this is the basic guide and additional configuration can cause confusion to the reader

Reply:
Doesn't seem to be used. I will remove it, as this is the basic guide and additional configuration can cause confusion to the reader